examples/with-redux/components/AddCount.js

```diff
@@ -0,0 +1,39 @@
+import React, {Component} from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { addCount } from '../store'
+
+class AddCount extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.add = this.add.bind(this)
+  }
+
+  add = () => {
+    this.props.addCount()
+  }
+
+  render () {
+    const { count } = this.props
+    return (
+      <div style={{paddingBottom: 20 + 'px'}}>
+        <h1>AddCount: <span>{count}</span></h1>
+        <button onClick={this.add}>Add To Count</button>
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = (state, ownProps) => {
```

Comment:
Thanks for reviewing all the code @impronunciable. After I make these changes, what's the best way to update this pull request with the updated code? I tried to do it the first time when I fixed all the linting, but it made a second pull request as you might have noticed and looks like closed :)

Cheers,
Spencer